pages/landing/enterprise-eks/pricing/index.html

```diff
@@ -0,0 +1,125 @@
+      - name: Kubernetes Native CI/CD
+        description: "Deploy your apps using GitOps practices with Kubernetes native tooling. Employ techniques such as canary deployments, feature flags, and staggered deployments, with built-in notifications and alerts. Examples: Flux, Argo CD."
+        plans: ["Standard", "Early Access"]
+      - name: Self-service Dashboard
+        description: "Have your team deploy apps using self-service dashboards that track deployment progress and provide error messages right in the UI. Examples: Argo CD Dashboard, Lens."
+        plans: ["Standard", "Early Access"]
+      - name: Metrics Aggregation
+        description: "Instrument your apps and ship metrics to metrics aggregation services. Query metrics along multiple dimensions and create dashboards to view app behavior. Examples: Prometheus, Datadog, CloudWatch."
+        plans: ["Standard", "Early Access"]
+      - name: Log Aggregation
+        description: "Ship all container logs to a central location. Query, inspect, and stream all logs from a single location. Examples: Datadog, Elastic Stack, CloudWatch."
+        plans: ["Standard", "Early Access"]
+      - name: Pluggable deployment strategies
+        description: Deploy your apps using a variety of industry standard deployment techniques, including canary releases and blue/green deploys. Institute automatic rollbacks in the event of a deployment failure or error condition.
+        plans: ["Standard", "Early Access"]
+      - name: Cloud-native Reference Applications
+        description: "Complete reference applications demonstrating how to deploy and optimize various applications.  Examples: Spring boot, Ruby on Rails, Django."
+        plans: ["Standard", "Early Access"]
+      - name: Gruntwork Patcher
+        description: Keep your apps and infrastructure always up to date. Early Access subscribers will also have early access to Gruntwork Patcher.
+        plans: ["Standard", "Early Access"]
+  - title: Support
+    features:
+      - name: Commercial maintenance and support
+        description: All infrastructure modules receive continuous updates that are commercially maintained by a team of DevOps experts.
+        plans: ["Standard", "Early Access"]
+      - name: Community forum
+        description: Access to a community Slack channel where Gruntwork customers can ask for assistance from other community members and Gruntwork engineers
+        plans: ["Standard", "Early Access"]
+      - name: Private shared Slack Channel
+        description: A dedicated Slack channel for fast, private access to support from Gruntwork engineers
+        plans: ["Early Access"]
+      - name: SLA on response times
+        description: Gruntwork will provide SLAs to guarantee you the fastest support possible
+        plans: ["Early Access"]
+      - name: Prioritized bug fixes
+        description: Gruntwork engineers will prioritize your bug fixes
+        plans: ["Early Access"]
+
+pricing:
```

Comment:
NIT: for some reason there is now a horizontal scroll bar on the pricing table that shouldn't be there. We should file a bug to tweak the CSS and fix that.

Reply:
Filed https://github.com/gruntwork-io/gruntwork-io.github.io/issues/573